stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -137,6 +140,103 @@ fn advance_to_nakamoto(
     // peer is at the start of cycle 8
 }
 
+/// Bring a TestPeer into the Nakamoto Epoch
+fn advance_to_nakamoto_long(
```

Comment:
I'm curious why this is needed, given that we already have `make_nakamoto_peer_from_inv()`?

Reply:
Firstly, because I don't want to use the test plan interface (it would be incompatible with testing invalid blocks).

Second, it invokes `net::tests::advance_to_nakamoto` (rather than `nakamoto::coordinator::tests::advance_to_nakamoto`). I didn't know which one of those to use -- neither work off the shelf here -- both construct a PoX reward set where everyone stacks to the same address. I could probably modify `net::tests::advance_to_nakamoto` to not do that, and it seems like maybe it would be what I wanted? I'm not sure why its twice as long as the other functions, but I could try to use it instead.

Reply:
Updated in 41a723c56